css/02-elements.css

```diff
@@ -35,10 +35,6 @@ h1, h2, h3, h4, h5, h6 {
 	font-weight: 700;
 }
 
-.wf-active .site-branding {
```

Comment:
Probably leave this here - the other one is specifically targeting menu links, and there could be an instance where there is something in the header that is not a menu link e.g. a tagline or something in the future.

Reply:
👍